installer/dbscript/dbscript-2.sql

```diff
@@ -4417,3 +4417,23 @@ VALUES (1, 0, 1, 0, 0, @apiv2_attendance_configuration_data_group_id, @admin_rol
        (0, 1, 1, 0, 1, @apiv2_attendance_my_attendance_record_data_group_id, @ess_role_id),
        (0, 1, 1, 0, 0, @apiv2_attendance_employee_attendance_record_data_group_id, @admin_role_id),
        (0, 1, 1, 0, 0, @apiv2_attendance_employee_attendance_record_data_group_id, @supervisor_role_id);
+
+ALTER TABLE `ohrm_leave` DROP `comments`;
+ALTER TABLE `ohrm_leave_request` DROP `comments`;
+
+-- Maintenance Module
+
+INSERT INTO ohrm_data_group (`name`, `description`, `can_read`, `can_create`, `can_update`, `can_delete`)
+VALUES ('apiv2_purge', 'API-v2 Maintenance - Purge Employee', 1, 0, 0, 1);
```

Comment:
will go with this pattern, since we have to add candidate purge also. 
you have to change variables accordingly
```suggestion
VALUES ('apiv2_purge_employee', 'API-v2 Maintenance - Purge Employee', 1, 0, 0, 1);
```

Reply:
Changed